state_version_output.go

```diff
@@ -16,6 +16,7 @@ var _ StateVersionOutputs = (*stateVersionOutputs)(nil)
 // TFE API docs: https://www.terraform.io/docs/cloud/api/state-version-outputs.html
 type StateVersionOutputs interface {
 	Read(ctx context.Context, outputID string) (*StateVersionOutput, error)
+	ReadCurrent(ctx context.Context, workspaceID string) (*StateVersionOutputsList, error)
```

Comment:
I see we have this precedence for ReadCurrent and ReadCurrentwithOptions: https://github.com/hashicorp/go-tfe/blob/main/state_version.go#L33-L36 But if this endpoint won't be adding options in the near future, then we are good.